About Section: Add a page about contributing and APG TF membership

mcking65 opened the pull request:
Resolve #2706  by adding a page to the about section about contributing.

### Preview links

* [Preview revised about page](https://deploy-preview-236--aria-practices.netlify.app/aria/apg/about/)
* [Preview new contributing page](https://deploy-preview-236--aria-practices.netlify.app/aria/apg/about/contributing/)

___
[WAI Preview Link](https://deploy-preview-236--aria-practices.netlify.app/ARIA/apg) _(Last built on Mon, 02 Oct 2023 19:57:43 GMT)._

shirsha commented:
Please add "Back to top" a landmark role. Discussed in the apg meeting and this will be deleted as it is part of template

The "[W3C Web Accessibility Initiative (WAI)](https://www.w3.org/WAI/)" visually look as heading but not marked up as such

daniel-montalvo commented on content/about/contributing/contributing.html:
```diff
@@ -0,0 +1,132 @@
+      <h2>Providing feedback</h2>
+      <p>
+        The APG provides two public feedback channels -- GitHub and email.
+        Links to both are provided at the bottom of every page in the APG.
```
```suggestion
        Links to both are provided at the bottom of every APG page.
```

mcking65 replied:
Left unchanged because so the emphasis is on "every page". "In the APG" is almost superfluous, but I've chosen to leave it because the claim might not be consistently true across all WAI pages.

daniel-montalvo replied:
It should be, but I don't feel strongly anyway.

daniel-montalvo commented on content/about/contributing/contributing.html:
```diff
@@ -0,0 +1,132 @@
+      <p>
+        The task force also uses the public aria-practices mailing list for email discussion.
+        Meeting announcements, agendas, and links to minutes are sent to the mailing list.
+        While GitHub issues are the preferred place to discuss APG content, the mailing list is available to anyone who would prefer to communicate with the APG Task Force by
```
```suggestion
        While GitHub issues are the preferred place to discuss APG content, the mailing list is available to anyone who would prefer to communicate by
```

daniel-montalvo commented on content/about/contributing/contributing.html:
```diff
@@ -0,0 +1,132 @@
+        The task force also uses the public aria-practices mailing list for email discussion.
+        Meeting announcements, agendas, and links to minutes are sent to the mailing list.
+        While GitHub issues are the preferred place to discuss APG content, the mailing list is available to anyone who would prefer to communicate with the APG Task Force by
+        <a href="mailto://public-aria-practices@w3.org">sending email</a>.
```
```suggestion
        <a href="mailto://public-aria-practices@w3.org">sending email to the APG Task Force</a>.
```

daniel-montalvo commented on content/about/contributing/contributing.html:
```diff
@@ -0,0 +1,132 @@
+      </p>
+    </section>
+    
+    <section id="making-content-improvements">
```
```suggestion
    <section id="Making-improvements-to-documentation-or-code">
```

Text to lowercase and hyphenated

daniel-montalvo commented on content/about/contributing/contributing.html:
```diff
@@ -0,0 +1,132 @@
+      </ol>
+      <p>
+        If you run into any difficulties, please feel free to ask questions either via a GitHub issue or by
+        <a href="mailto://public-aria-practices@w3.org">sending email to the public authoring practices mailing list</a>.
```
```suggestion
        <a href="mailto://public-aria-practices@w3.org">sending email to the ARIA Authoring Practices public mailing list</a>.
```

daniel-montalvo commented on content/about/contributing/contributing.html:
```diff
@@ -0,0 +1,132 @@
+      </p>
+    </section>
+
+    <section id="joining-task-force">
```
```suggestion
    <section id="joining-the-authoring-practices-task-force">
```

daniel-montalvo commented on content/about/contributing/contributing.html:
```diff
@@ -0,0 +1,132 @@
+        <a target="_blank" href="https://www.w3.org/WAI/ARIA/task-forces/practices/">ARIA Authoring Practices Task Force</a>,
+        which  is a sub-group of the
+        <a target="_blank" href="https://www.w3.org/WAI/ARIA/">ARIA Working Group</a>.
+        The task force meets weekly to coordinate work on development of the APG.
```
```suggestion
        The Task Force meets weekly to coordinate work on development of the APG.
```

daniel-montalvo commented on content/about/contributing/contributing.html:
```diff
@@ -0,0 +1,132 @@
+        which  is a sub-group of the
+        <a target="_blank" href="https://www.w3.org/WAI/ARIA/">ARIA Working Group</a>.
+        The task force meets weekly to coordinate work on development of the APG.
+        Task force participants are expected to regularly attend meetings and to actively contribute.
```
```suggestion
        Task Force participants are expected to regularly attend meetings and to actively contribute.
```

daniel-montalvo commented on content/about/contributing/contributing.html:
```diff
@@ -0,0 +1,132 @@
+      </p>
+    </section>
+
+    <section id="aria-at">
```
```suggestion
    <section id="Helping-provide-assistive-technology-support-data">
```

daniel-montalvo reviewed (changes requested):
Thanks for putting all of this together. Minor editorial suggestions.

I would focus mainly on the ids with text to lowercase and hyphenated, for consistency and better predictability of anchor links. I may have skipped some of these

andreancardona reviewed (approved):
Other than Daniel's comments looks good to me!

mcking65 commented on content/about/contributing/contributing.html:
```diff
@@ -0,0 +1,132 @@
+  <main>
+    <h1>Contributing to the APG</h1>
+
+    <section id="intro">
```
I commited a similar change. The id you suggested had a misspelling.

mcking65 commented on content/about/contributing/contributing.html:
```diff
@@ -0,0 +1,132 @@
+    <section id="intro">
+      <h2>Introduction  </h2>
+      <p>
+        Continuous improvement to the utility and quality of the APG depends on broad community input and participation.
```
I made a similar commit. Your suggestion didn't include the word "Guide".

mcking65 commented:
@shirsha wrote:

> Please add "Back to top" a landmark role. Discussed in the apg meeting and this will be deleted as it is part of template

I don't know if it will be deleted; I would like it to be deleted. However, I don't know if there is a template issue open about it. That'd take a bit of searching.

> The "[W3C Web Accessibility Initiative (WAI)](https://www.w3.org/WAI/)" visually look as heading but not marked up as such

The styling might make it appear to be a heading, but semantically it does not serve as a heading. It is just a prominent link in the banner. It is really important to only use heading elements where the element is semantically serving the purpose of a heading, not just styled in a way that is similar to a heading.

mcking65 commented:
@daniel-montalvo

A quick note on capitalization of the term "task force" since it came up in your review.

I have capitalized the term when it is a proper noun and used lowercase otherwise. That is, when writing out the full name "ARIA Authoring Practices Task Force", the words "Task" and "Force" are capitalized. If the words "task force" appear separate from the full name of the task force, they are not capitalized. I am fairly confident this is consistent with the Chicago manual of style referenced by the editorial guidelines. If there is something else I should consider, please let me know.

daniel-montalvo commented:
> @daniel-montalvo
> 
> A quick note on capitalization of the term "task force" since it came up in your review.
> 
> I have capitalized the term when it is a proper noun and used lowercase otherwise. That is, when writing out the full name "ARIA Authoring Practices Task Force", the words "Task" and "Force" are capitalized. If the words "task force" appear separate from the full name of the task force, they are not capitalized. I am fairly confident this is consistent with the Chicago manual of style referenced by the editorial guidelines. If there is something else I should consider, please let me know.

I'm OK as long as we are consistent. What you suggest seems consistent to me.

daniel-montalvo reviewed (approved):
Approving from an editorial perspective. Note that link-checker still flags a broken link  on contributing.html line 69

a11ydoer commented:
In New contributing page, sending email to APG has two unnecessary slash, //public-aria-practices@w3.org [undefined://public-aria-practices@w3.org] and we can update that in a correct form.

a11ydoer reviewed (changes requested):
mailto code should remove two slashes, "//" in front of mailing address.

a11ydoer reviewed (approved):
I fixed mailto code and directly commited the change to speed up the publication. Please let me know if you prefer to creat PR instead despite that this is a minor update, @mcking65

mcking65 commented:
Thank you @a11ydoer for catching that. BTW, the links work with the slashes, at least in Chrome, but they shouldn't be there.

> I fixed mailto code and directly commited the change to speed up the publication.

There were two; I fixed the other.

>Please let me know if you prefer to creat PR instead despite that this is a minor update, @mcking65

Definitely not a PR for minor changes. Using the suggestions feature in a review is the most straightforward way to add code and ensure everyone is aligned, but for something like this where there is no question about it, pushing a commit is fine.

mcking65 commented:
Thank you @a11ydoer and @daniel-montalvo! @howard-e has updated the link-checker in #2812, and I'm just waiting on a review from @alflennik  on that. Then, if all is good, I will merge the link-checker changes. Then, the tests on this should all be passing, and I will merge it.

a11ydoer commented:
> Thank you @a11ydoer for catching that. BTW, the links work with the slashes, at least in Chrome, but they shouldn't be there.
> 
> > I fixed mailto code and directly commited the change to speed up the publication.
> 
> There were two; I fixed the other.
> 
> > Please let me know if you prefer to creat PR instead despite that this is a minor update, @mcking65
> 
> Definitely not a PR for minor changes. Using the suggestions feature in a review is the most straightforward way to add code and ensure everyone is aligned, but for something like this where there is no question about it, pushing a commit is fine.

Thanks for fixing the error and your feedback about using "suggestions feature"
btw, I tested sending the email with two slashes and outlook reported me the error.

shirsha commented:
Thanks, Matt !!. Changed changes are reviewed
